aixplain/modules/model/__init__.py

```diff
@@ -204,21 +204,21 @@ def run(
             Dict: parsed output from model
         """
         start = time.time()
-        try:
-            response = self.run_async(data, name=name, parameters=parameters)
-            if response["status"] == "FAILED":
+        payload = build_payload(data=data, parameters=parameters)
+        url = f"{self.url}/api/v2/execute/{self.id}"
+        logging.debug(f"Model Run Sync: Start service for {name} - {url}")
+        response = call_run_endpoint(payload=payload, url=url, api_key=self.api_key)
+        if response["status"] == "IN_PROGRESS":
```

Comment:
Never assume that status will always be there. In case something horribly goes wrong, it wont. Better have the sdk handle it gracefully than crashing, no?

Reply:
In `call_run_endpoint` I make sure a status is always returned.